README.md

```diff
@@ -57,6 +57,26 @@ func TestSomething(t *testing.T) {
 }
 ```
 
+Spin up a replica set server:
+
+```go
+var mongoServer memongo.Server;
+
+func TestMain(m *testing.M) {
+  mongoServer, err = memongo.StartWitOptions(&memongo.Options{MongoVersion: "4.2.1", ShouldUseReplica: true})
```

Comment:
Can we call this `ConfigureReplicaSet`? I think you need to be a little clearer in this documentation that this just configures the single server as a replica set -- but does not set up a secondary or actually perform replication. I'd suggest being pretty explicit here and say something like "This configures the Mongo server to be a replica set, and calls `rs.initiate()` after startup. It still only runs a single Mongo server."